website/content/en/docs/concepts/scheduling.md

```diff
@@ -260,6 +260,14 @@ If they all fail, Karpenter will fail to provision the pod.
 Karpenter will backoff and retry over time.
 So if capacity becomes available, it will schedule the pod without user intervention.
 
+#### Preferred Affinities
+
+Karpenter treats preferred affinities as hard affinities in almost all circumstances.
+
+Karpenter interprets preferred affinities as a soft affinity when building up topology requirements for scheduling to a node. Preferred affinities are ignored in regards to topology zones and hard affinities should be used as documented above.
+
+In all other circumstances, Karpenter treats the required and preferred affinities as if they were required affinities. When determining if a pod can schedule on a node (absent topology requirements), or if a pod can be shifted to a new node, soft affinities are treated as if hard affinities. This includes when determining if a node can be safely disrupted, a node with pods that contain unsatisfied preferred affinities will not be disrupted if there are no matching nodes that meet it's preferred requirements.
```

Comment:
There's a caveat to this statement. Specifically, if we can't satisfy the hard affinity in any way, a preference will cause us to fallback from treating the affinity as strictly required to removing it altogether.

Reply:
Updated to include this language.